json-lint/schemas/inventory/adservers/emptyad.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$id": "#emptyAd",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "no AD",
+    "description": "AdServer Implemented to load empty adServer instead of custom",
```

Comment:
Just to clearer I would write something like `Empty ad slot`

Reply:
yup